py/mpconfig.h

```diff
@@ -1649,6 +1669,13 @@ typedef double mp_float_t;
 #endif
 #endif
 
+// Explicitly annotate switch case fall throughs
+#if defined(__GNUC__) && __GNUC__ >= 7
+#define MP_FALLTHROUGH __attribute__((fallthrough));
+#else
+#define MP_FALLTHROUGH
+#endif
+
```

Comment:
More stragglers in "bitops", for some reason (it was my fault)

tannewt/merge_1.14:shared-module/bitops/__init__.c:            FALLTHROUGH;